### DISCUSSION TITLE
HowTo assembly the PCB / Tag Reader V 4.1

### DISCUSSION THREAD

**Discussion by Noschvie** [adonno/tagreader#117] [Q&A] [answered]
Hello
do you have a diagramm showing how to put the PN532 and the D1 mini to the PCB "Tag Reader V 4.1" ?
Which one is upside / downside ?
Thanks!
Norbert

**Comment by adonno** [comment, marked as answer]
hi there should be a youtube video available upon scanning the QR code 

https://youtu.be/vknJsM6G7ww
